packages/server/src/websockets/builder.ts

```diff
@@ -14,9 +14,9 @@ export default class BuilderSocket extends BaseSocket {
     super(app, server, "/socket/builder", [authorized(permissions.BUILDER)])
   }
 
-  async onConnect(socket: Socket) {
+  async onConnect(socket?: Socket) {
```

Comment:
This should probably be handled at the caller level. Will it ever need to be called `onConnect` without a socket?

Reply:
I didn't write this - just fixed the typescript definition!